pkg/api/soc.go

```diff
@@ -105,8 +105,22 @@ func (s *server) socUploadHandler(w http.ResponseWriter, r *http.Request) {
 		return
 
 	}
+
 	ctx := r.Context()
 
+	has, err := s.storer.Has(ctx, chunk.Address())
```

Comment:
why not check in swarm why only locally?

Reply:
so uploading a chunk to the API might take up to a minute for all requests to time-out and conclude that the chunk doesn't exist. I don't find this so user-friendly. Can make the change but meh